iothub/device/tests/Authentication/ClientAuthenticationWithTokenRefreshTests.cs

```diff
@@ -240,6 +240,47 @@ public async Task ClientAuthenticationWithSakRefresh_WithModule_SharedAccessKeyC
             Assert.AreEqual(token1, token2.TokenValue);
         }
 
+        [TestMethod]
+        [ExpectedException(typeof(FormatException))]
+        public void ClientAuthenticationWithSakRefresh_InvalidKey_Throws()
+        {
+            var builder = new SharedAccessSignatureBuilder()
```

Comment:
FYI when initializing a class with an empty ctor and an initializer list, you can omit the parens.

```suggestion
            var builder = new SharedAccessSignatureBuilder
```